python/makecirc.py

```diff
@@ -0,0 +1,151 @@
+from lxml import etree
+
+
+E = etree.Element
+
+def makexml(name, nsl):
+    xsi = "http://www.w3.org/2001/XMLSchema-instance"
+    ns = {"xsi": xsi}
+    attr = {"{%s}noNamespaceSchemaLocation" % xsi: nsl}
+    t = E(name, attrib=attr, nsmap=ns)
+    return t
+
+def printxml(t, fn):
+    etree.ElementTree(t).write(fn, pretty_print=True, encoding='UTF-8', xml_declaration=True) 
+
+def makenet(base, length, lanes, maxSpeed=40):
+    import subprocess
+    import sys
+
+    name = "%s-%dm-%dl" % (base, length, lanes)
+
+    nodfn = "%s.nod.xml" % name
+    edgfn = "%s.edg.xml" % name
+    typfn = "%s.typ.xml" % name
+    cfgfn = "%s.netccfg" % name
+    netfn = "%s.net.xml" % name
+    l4 = length/4.
+
+    x = makexml("nodes", "http://sumo.dlr.de/xsd/nodes_file.xsd")
+    x.append(E("node", id="bottom-left", x=repr(0), y=repr(0)))
+    x.append(E("node", id="bottom-right", x=repr(l4), y=repr(0)))
+    x.append(E("node", id="top-right", x=repr(l4), y=repr(l4)))
+    x.append(E("node", id="top-left", x=repr(0), y=repr(l4)))
+    printxml(x, nodfn)
+
+    x = makexml("edges", "http://sumo.dlr.de/xsd/edges_file.xsd")
```

Comment:
So the `circular` networks are square networks? Do we know if there are any consequences for our simulations / models at the corners of the network, i.e. is there anything different driving-wise about a corner as opposed to a straight road?